### ISSUE TITLE
EPIC: Prepare the Communities of Practice page for Homepage Launch

### ISSUE THREAD

**Issue by priyanka02art** [hackforla/website#7671]
### Overview
We need to track existing issues and make new issues to deliver Communities of Practice page complete enough for the Homepage launch

#### Details
We need to track and support the final wrap up items for the homepage launch, so that it can be completed.

### Action Items
- [x] List all the issues related to the label
- [x] Identify what open issues have to be resolved before launch, by classifying the issues into the following categories using MoSCoW method in a comment.
   - Must Have
   - Should Have
   - Could Have
   - Won't Have this time
- [x] Add the following details under each issue
   ```
   - why: [ADD WHY ITS IMPORTANT TO TO THE LAUNCH]
   - status: [ADD STATUS e.g., Icebox, Emergent Requests, New Issue Approval, ERs and epics that are ready to be turned into issues, Prioritized Backlog, In Progress (actively working), Questions / In Review]
      - draft: [true OR false]
      - responsible: [ADD GITHUB HANDLES WITHOUT @]
   - role: [ADD ROLE e.g. data scientist, infrastructure, design, design lead, user research, Research Lead, dev leads, data analyst, BA, hfla leadership, back end/devOps, writing, front end, merge team, technical writer]
   ```
- [x] Once you receive sign-off from Staff PM add to `## Relevance of the issues to the Homepage launch`
- [x] Review comments with `Notes carried forward from epic` if it exists, and make sure all previous items are addressed in the top of the issue.
   - If it specifies new issues to make then add explicit action items for making those issue in a new comment on this issue
      - get sign-off on those new issues to make by Staff PM 
- [x] Identify items that need issues opened by looking at the page and features and seeing what is not already addressed by the existing issues or new issues to make list.
- [x] Make the issues specified
- [x] Link them as sub issues to this issue
- [x] Classify them using MoSCoW and add the details (why, status, draft, responsible, role)
- [ ] Get Sign-off and prioritization of the issues by Staff PM
- [ ] If this page has content that must be checked for accuracy before homepage launch, add the detail to the epic for tracking that (resource 1.05)

### Resources/Instructions
- Parent Epic: https://github.com/hackforla/website/issues/2481

#### Used for creating this issue
- 1.01 Label name: P-Feature: Communities of Practice
    - 1.01.01 Issue Tab: [P-Feature: Communities of Practice](https://github.com/hackforla/website/issues?q=state%3Aopen%20label%3A%22P-Feature%3A%20Communities%20of%20Practice%22)
    - 1.01.02 Project Board: [P-Feature: Communities of Practice](https://github.com/orgs/hackforla/projects/86/views/1?filterQuery=label%3A%22P-Feature%3A+Communities+of+Practice%22+)
- 1.02 URL of page on Hack for LA website: https://www.hackforla.org/communities-of-practice
- 1.03 URL of link to code search: [repo:hackforla/website NOT language:SVG NOT path:/^_data/internal/credits// Communities of Practice](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+NOT+language%3ASVG+NOT+path%3A%2F%5E_data%5C%2Finternal%5C%2Fcredits%5C%2F%2F+Practice+page&type=code)
- 1.04 URL of WIKI page: https://github.com/search?q=repo%3Ahackforla%2Fwebsite+Communities+of+Practice&type=wikis
   - 1.04.01 URL of future edits to wiki pages: https://github.com/hackforla/website/wiki/How-to-Contribute-to-the-Wiki (check this page to see if there are any other new edits in the queue.)
- 1.05 Issue for adding items to a final checklist: #8417

#### Created by this issue
- 2.01 [EXAMPLE would be a link to an issue related to delivering this page]
---
### Must Have
TBD

### Should Have
TBD

### Could Have
TBD

---
## Relevance of the issues to the Homepage launch

last updated: 2026-04-21
Highest number found: 8540

### Must Have
- [ ] #7671   
   - why: It is the epic for this page (this issue)
   - status: In Progress
      - draft: False
      - responsible: kimberlyvaldez07 
   - role: Product

- [ ] #8540
   - why: We need to keep project information up to date so that visitors to the website can find accurate information.
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end 

- [ ] #8511
   - why: Our website needs to use the Oxford comma consistently so that the website is correct and uniform. In this issue we will be changing text on the Communities of Practice page.
   - status: Ready for Prioritization 
      - draft: False
      - responsible: None 
   - role: Back end/devOps and front end

- [ ] #8325
   - why: PMs can more efficiently get update requests made into developer issues
   - status: new issue approval 
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps, merge team, front end 

### Should Have
- [ ] #7420
   - why: Need to fix spelling
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: front end

### Could Have
<details><summary>List of all could have issues</summary>

</details> 

### Won't have this time
<details><summary>List of all won't have issues</summary>

- [ ] #6161 

</details> 

#### Templates
```
- [ ] #
   - why: 
   - status: 
      - draft: 
      - responsible: 
   - role:  
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

<details><summary>List of all issue by number</summary>

- #8540
- #8511
- #8325
- #7671
- #7420
- #6161

</details>

**Comment by priyanka02art** [comment]
- URL: https://www.hackforla.org/communities-of-practice

- [ ] [Communities of Practice Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+communities+of+practice%22)
          - [ ] [Needs to be reviewed to see if updates are required]

**Comment by priyanka02art** [comment]
# List of all issues
- [ ] #7671   
- [x] #7905 
- [x] #7087 
- [x] #7111   


## Relevance of the issues to the Homepage launch
### Must Have
- [ ] #7671   
   - why: It is the epic for this page (this issue)
   - status: New Issue Approval
      - draft: True
      - responsible: None
   - role: Product
- [x] #7905 
   - why: Need to have Communities and Practice page with correct information.
   - status: New Issue Approval
      - draft: False
      - responsible: None
   - role: Product
- [x] #7087  
   - why: Need to have Communities and Practice page with correct information.
   - status: Ice Box
      - draft: False
      - responsible: None
   - role: Product

### Should Have
- [x] #7111  
   - why: Refactor to fix spelling
   - status: Closed
      - draft: False
      - responsible: @daras-cu 
   - role: Front end 

### Could Have


### Won't have this time

#### Template
```
   - why: 
   - status: 
      - draft
      - responsible: 
   - role: 
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

**Comment by priyanka02art** [comment]
Issues to triage 
 
- #7520
   - why: 
   - status: 
      - draft
      - responsible: 
   - role:

**Comment by ExperimentsInHonesty** [comment]
@charmainemak09 Please provide update

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

You can use this template
```
1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary): 
```

**Comment by priyanka02art** [comment]
Sent email to Charmaine on 2025-06-05

> Hi Charmaine,
> 
> Please review the issues assigned to you and provide a brief update on each one by asap. Specifically:
> 
> Current status (e.g., in progress, blocked, completed)
> 
> Any dependencies or blockers
> 
> Estimated timeline for completion (if not yet done)
> 
> You can leave your updates directly as comments on the respective issues so everything stays in one place.
> 
> Let me know if you have any questions or need help unblocking anything.
> 
> Thanks,
> Priyanka

**Comment by priyanka02art** [comment]
### Issues list associated with this epic
last updated: 2025-08-24
Highest number found: 8286
- #8286
- #8251  
- #8120
- #8119
- #8040
- #7988
- #7420

---
### Must have
- #8286 
   - why: Need to have Communities and Practice page with correct information.
   - status: New Issue Approval
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: Product

- #8251 
   - why: Need to have Communities and Practice page with correct information.
   - status: New issue approval
      - draft: False
      - responsible: None
   - role: Product

- #8120 
   - why: Need to have Communities and Practice page with correct information.
   - status: New issue approval
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #8119
   - why: Need to have Communities and Practice page with correct information.
   - status: Prioritized Backlog
      - draft: False
      - responsible: None
   - role: back end/devOps and front end

- #8040
   - why: Need to have Communities and Practice page with correct information.
   - status: New issue approval
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: product

- #7988
   - why: Enhance user experience
   - status: In progress (actively working)
      - draft: False
      - responsible: @kdaca19xx
   - role: front end
  
### Should have
- #7420
   - why: Need to fix spelling
   - status: Prioritized backlog
      - draft: False
      - responsible: None
   - role: front end
### Could have

### Won't have

**Comment by kimberlyvaldez07** [comment]
### Issues list associated with this epic
last updated: 2025-09-24
Highest number found: 8325

- #8325 

---

### Must Have 

- #8325
   - why: PMs can more efficiently get update requests made into developer issues
   - status: new issue approval 
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: back end/devOps, merge team, front end 
   
### Should Have 
 
### Could Have

### Wont Have

**Comment by kimberlyvaldez07** [comment]
2026-08-18

Using this search 
[state:open label:"P-Feature: Communities of Practice" is:issue](https://github.com/hackforla/website/issues?q=state%3Aopen%20label%3A%22P-Feature%3A%20Communities%20of%20Practice%22%20is%3Aissue) 

This is the highest issue number found 8511 it is addressed at the top of this issue.
